ddcz/views.py

```diff
@@ -302,4 +311,5 @@ def phorum(request):
 
     return render(request, 'discussions/phorum-list.html', {
         'discussions': discussions,
-    })
+        'phorumCommentaryForm': PhorumCommentaryForm()
+    })
```

Comment:
Missing newline—please configure your editor to add it

Reply:
New line added